app/src/main/kotlin/com/wire/android/mapper/message/content/text/MessageTextContentMapper.kt

```diff
@@ -0,0 +1,80 @@
+package com.wire.android.mapper.message.content.text
+
+import com.wire.android.R
+import com.wire.android.mapper.message.content.MessageResourceProvider
+import com.wire.android.model.ImageAsset
+import com.wire.android.ui.home.conversations.model.AttachmentType
+import com.wire.android.ui.home.conversations.model.MessageBody
+import com.wire.android.ui.home.conversations.model.QuotedMessageUIData
+import com.wire.android.ui.home.conversations.model.UIMessageContent
+import com.wire.android.util.time.ISOFormatter
+import com.wire.android.util.ui.UIText
+import com.wire.android.util.ui.WireSessionImageLoader
+import com.wire.kalium.logic.data.id.ConversationId
+import com.wire.kalium.logic.data.message.MessageContent
+import javax.inject.Inject
+
+class MessageTextContentMapper @Inject constructor(
```

Comment:
MessageTextContent is also not an 2 lines of code beast that is why I splitted it into a seperate class too